app/experimenter/nimbus-ui/src/components/PageNew/index.test.tsx

```diff
@@ -40,33 +96,67 @@ describe("PageNew", () => {
     expect(global.console.log).toHaveBeenCalledWith("CANCEL TBD");
   });
 
-  const Subject = () => {
-    return <PageNew />;
+  const mkGqlMocks = ({
+    message = "success" as string | Record<string, any>,
+    status = 200,
+    nimbusExperiment = { ...mockSubmit, slug: "foo-bar-baz" },
+  } = {}) => [
+    {
+      request: {
+        query: CREATE_EXPERIMENT_MUTATION,
+        variables: {
+          input: mockSubmit,
+        },
+      },
+      result: {
+        errors: undefined as undefined | any[],
```

Comment:
What's `undefined as undefined` doing?

Reply:
Not just `undefined as undefined` - the `| any[]` also means that the result type for the function allows undefined or an array for errors. Did this inline rather than define a separate return type for `mkGqlMocks`

Reply:
Oh yeah, and the reason for this typing here is that it enables [this other line](https://github.com/mozilla/experimenter/pull/3791/files#diff-b489135824d87fb1bef4f7cb4c9258c8bac80d980a2321f80a3edf2c8c5929bcR84):
```
gqlMocks[0].result.errors = [new Error(expectedError)]
```